virtkvm/__init__.py

```diff
@@ -107,14 +113,22 @@ def _call_dccutil(display: dict, ident: int):
             "setvcp", hex(display["feature"]), hex(ident)
         ])
 
+    @staticmethod
+    def _call_commands(command: str):
+        return subprocess.call(command.split(" "), shell=True)
```

Comment:
Now that ``shell=True``, the command should not be split.

Reply:
Okay, now I understand the difference with the parameter `shell`.

Thanks for the review and the advises :)